journal-of-water-sanitation-and-hygiene-for-development.csl

```diff
@@ -0,0 +1,604 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB" demote-non-dropping-particle="never">
+  <info>
+    <title>Journal of Water Sanitation and Hygiene for Development</title>
+    <title-short>JWaSHDev</title-short>
+    <id>http://www.zotero.org/styles/journal-of-water-sanitation-and-hygiene-for-development</id>
+    <link href="http://www.zotero.org/styles/journal-of-water-sanitation-and-hygiene-for-development" rel="self" />
+    <link href="http://www.zotero.org/styles/apa-old-doi-prefix" rel="template" />
```

Comment:
I swear I saw it somewhere and made that choice specifically, but of course I can't find it now. Let me do some digging and come back here or make the changes.